docs/source/deploying/workers.rst

```diff
@@ -0,0 +1,58 @@
+Task Workers
+=============
+
+Quetz offers 3 types of backends for task workers. Each of them is explained below.
+
+Thread
+-----------
+
+Thread workers process tasks in a separate thread. This functionality is in-built into FastAPI using
+``BackgroundTasks`` (https://fastapi.tiangolo.com/tutorial/background-tasks/).
+
+
+Subprocess
+-----------
+
+Subprocess workers start in a separate process and are implemented through ``ProcessPoolExecutor`` of the
+``concurrent.futures`` module. Once again, this is shipped as a part of Quetz.
+
+
+Redis
+-----------
+
+For advanced use-cases, Quetz also offers the ability to use ``redis-queue`` to manage jobs and run them on
+multiple processes or even multiple servers.
+
+To use this backend, one needs to setup ``redis`` and ``redis-queue``.
```

Comment:
since we are making redis-py and rq optional depenedencies you could also add here micromamba instructions to install it.

Reply:
We already gave a link to `instructions` for both of them.